libs/packages/sam-formly/src/lib/formly-reset/formly-reset.component.spec.ts

```diff
@@ -92,7 +92,10 @@ describe('SDS Formly Reset', () => {
     resetEl = fixture.nativeElement.querySelector('.usa-button');
     resetEl.click()
 
-    expect(testComp.model).toEqual({ flat: null, nested: { nestedInput: null } });
+    expect(testComp.model).toEqual({
```

Comment:
This test was updated to match the expected behavior in @ngx-formly/core@5.10.10. Based on ngx-formly/ngx-formly#2621, the expected behavior was that the form be returned to defaultValue rather than null on reset. As such, I have updated this test